spec/game_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+
```

Comment:
Solid coverage on initializing objects and all of their attributes. Can't think of any ways to be more thorough at this time.